frontend/src/utils/index.js

```diff
@@ -543,15 +543,33 @@ export function addKymaAddon (options) {
   shootAddonList.push(kymaAddon)
 }
 
-export function compileMarkdown (text) {
+export function textColorFromColor (color) {
+  const iteratee = value => /^(darken|lighten|accent)-\d$/.test(value) ? 'text--' + value : value + '--text'
+  return map(split(color, ' '), iteratee)
+}
+
+export function compileMarkdown (text, linkColor = 'cyan darken-2', transformToExternalLinks = true) {
   if (!text) {
     return undefined
   }
-  return DOMPurify.sanitize(marked(text, {
+  let html = DOMPurify.sanitize(marked(text, {
```

Comment:
maybe move sanitize to the very end to catch the unlikely event that unsafe code slips in via the color variable?